pkg/security/auth.go

```diff
@@ -136,14 +136,14 @@ func UserAuthCertHook(insecureMode bool, tlsState *tls.ConnectionState) (UserAut
 		// The client certificate should not be a tenant client type. For now just
 		// check that it doesn't have OU=Tenants. It would make sense to add
 		// explicit OU=Users to all client certificates and to check for match.
-		if IsTenantCertificate(tlsState.PeerCertificates[0]) {
```

Comment:
Not sure how to handle this section which explicitly throws an error if we have using a client cert for a tenant or if the username (root) does not match the name on the client certificate. I tried modifying the username from root to the tenant ID while setting up the SQL connection but then I ran into being prompted for a password again.